### PR TITLE
Dunst status

### PR DESCRIPTION
**Pull request by elshize**
Notification script for `dunst` using `dunstctl is-paused`. See README for more information.

### CODE REVIEW

File: polybar-scripts/dunst-status/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Dunst Status & Toggle
+
+This is a simple script that displays dunst's `is-paused` status.
+It will display a crossed out bell icon
+if the notifications are paused,
+or a regular bell otherwise.
+
+![](screenshots/on.png)
```
Comment (x70b1) [inline polybar-scripts/dunst-status/README.md:8]: Lets rename the screenshots to `1.png` and `2.png`.

File: polybar-scripts/dunst-status/dunst-status.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+ON=""
```
Comment (x70b1) [inline polybar-scripts/dunst-status/dunst-status.sh:3]: Replace them with `#` as described in the contributions.md.

File: polybar-scripts/dunst-status/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Dunst Status & Toggle
```
Comment (x70b1) [inline polybar-scripts/dunst-status/README.md:1]: We should rename that script and all paths to `info-dunst`.

File: polybar-scripts/dunst-status/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Module Configuration
+
+```
+[module/dunst]
+type = custom/script
+format = <label>
+label = %output%
+exec = /path/to/dunst-status.sh
+tail = true
```
Comment (x70b1) [inline polybar-scripts/dunst-status/README.md:31]: Why do you use the `tail = true` ?
You could also run that script with an interval of 1. But that will use a lot of resources. A higher interval would be better.

**Review by x70b1 — CHANGES_REQUESTED** [review]

File: polybar-scripts/dunst-status/README.md (hunk 4 above)
Comment (elshize) [reply polybar-scripts/dunst-status/README.md:31]: The reason for that is to have an immediate response once the status is changed. This is similar to the example in the polybar repo: https://github.com/polybar/polybar/wiki/Module:-script#examples

If you have just a one-shot script on an interval, the change to the status won't happen right away, only after the interval. Which might seem like it's not working or simply be annoying, especially with higher interval. That's why tail is `true`, and clicking on the status icon sends a signal to the process to update. As explained below in the README, it also allows for having a script that toggles the status from, say, a key shortcut. This way, as long as `dunstctl` isn't called manually, we have a seamless experience. If it is, then it still works, but with that up-to-1-second lag. I can add a comment suggesting increasing the interval in the script if one doesn't plan to use `dunstctl` manually to change the status.

**Pull request closed without merging**